src/main/plugins/org.dita.pdf2/cfg/common/vars/commonvariables.xml

```diff
@@ -145,6 +145,14 @@ See the accompanying LICENSE file for applicable license.
   <variable id="Step Format">1</variable>
   <variable id="Substep Number"><param ref-name="number"/>) </variable>
   <variable id="Substep Format">a</variable>
+  <variable id="Step Number 1"><param ref-name="number"/>. </variable>
```

Comment:
With these new depth based variables, should we deprecate the old variables on rows 144-147?

Reply:
Updated substep code to use Step Number 2 / Step Format 2, and marked those 4 original vars deprecated